### PR TITLE
WRN-19202: Added Sample for React 18 Features

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [ ] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added Sample for React 18 Features

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-19202

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by seunghoh — CHANGES_REQUESTED** [review]

File: sandstone/pattern-react18-new/src/views/Suspense/SkeletonPage.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+			</ImageItem>
+			<Spinner
+				style={{position: 'absolute', top: '50%', left: '50%', margin: '-63px 0 0 -39px'}}
+				transparent
+			/>
+		</>
+	);
+};
+
+for (let i = 0; i < 12; i++) {
```
[the remark this answers is not in the document]
Comment (daniel-stoian-lgp) [reply sandstone/pattern-react18-new/src/views/Suspense/SkeletonPage.js:27]: no, it's not needed. I will remove it

File: sandstone/pattern-react18-new/src/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import {createRoot} from 'react-dom';
```
[the remark this answers is not in the document]
Comment (daniel-stoian-lgp) [reply sandstone/pattern-react18-new/src/index.js:1]: it seems that it has changed recently. announcement on GH about this change was made on March 1st. Thank you for the warning

**Review by seunghoh — APPROVED** [review]
LGTM!

**Comment by hong6316** [comment]
For updating sandstone/ui samples` whole changes related react18, merge this PR in [feature/WRO-2831](https://github.com/enactjs/samples/tree/feature/WRO-2831) branch

**Review by hong6316 — APPROVED** [review]
LGTM

File: sandstone/pattern-react18-new/package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  "eslintConfig": {
+    "extends": "enact"
+  },
+  "eslintIgnore": [
+    "node_modules/*",
+    "build/*",
+    "dist/*"
+  ],
+  "dependencies": {
```
[the remark this answers is not in the document]
Comment (hong6316) [reply sandstone/pattern-react18-new/package.json:32]: Done

**Pull request merged**